src/firefly/js/util/PositionParser.js

```diff
@@ -406,20 +406,21 @@ var makePositionParser = function(helper) {
 
         if (s && array.length>0) {
             if (array.length===1 && matches(array[0],COMBINE_SYS)) {
-                if (array[0].toUpperCase().startsWith('EC')) {
+                const inCoord= array[0].toUpperCase()
+                if (inCoord.startsWith('EC') || inCoord.startsWith('EC_')) {
```

Comment:
The 2nd condition would never be reached.

Reply:
will fix